integration/presto/src/test/scala/org/apache/carbondata/presto/integrationtest/PrestoTestNonTransactionalTableFiles.scala

```diff
@@ -97,7 +97,7 @@ class PrestoTestNonTransactionalTableFiles extends FunSuiteLike with BeforeAndAf
       sortColumns: List[String]): Any = {
     val schema = new StringBuilder()
       .append("[ \n")
-      .append("   {\"NaMe\":\"string\"},\n")
+      .append("   {\"name\":\"varchar\"},\n")
```

Comment:
Instead of updating the existing column to varchar, add new varchar column type and insert the long string more than 32k and verify with testcase.

Reply:
Done. Updated the testcase.